.gitignore

```diff
@@ -38,6 +38,7 @@ docs/_build
 env
```

Comment:
`.venv` is another recommended way to create Python venvs. This wasn't in the ignore, much like the other envs you can see in the list above. 

It caught me out as I almost committed it. So putting it so stop other future developers.

Reply:
Ah okay, I meant more as it isn't related to the work - also sorry I should have been clearer, when I said 'file' I meant the .gitignore file in general rather than just the .venv bit. I normally try to only commit work in a PR that is related to the ticketed work, or I contain it in a separate commit or something, so it is clear

Reply:
I see you're point. I don't see the harm in adding this as it's not effecting the project and is just improving ways of working for developers. But I'm happy to take it out and put into a clean up PR which is needed. I think there is a ticket for that as the README.md is a mess.